### PR TITLE
Add auth page

### PR DESCRIPTION
**Pull request by VLeads**
Added Login & Signup page

### CODE REVIEW

**Review by Maianki — APPROVED** [review]
Looks good to merge.

File: components/Auth/login.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+      <div class="search-box">
+        <i class="fas fa-search"></i>
+        <input type="text" placeholder="Search" />
+      </div>
+      <ul class="nav-links">
+        <li>
+          <a href="#" class="btn btn-primary">Login</a>
+        </li>
+        <li>
+          <a href="../Wishlist/wishlist.html"
```
Comment (venkateshwarans) [inline components/Auth/login.html]: try to have folders' names with lowercase.

File: components/Cart/cart.css [hunk 2]
```diff
@@ -123,6 +124,7 @@ input:focus{
 
 .order{
     width: 100%;
+    padding: 1.5rem 1.8rem
```
Comment (venkateshwarans) [inline components/Cart/cart.css]: try to keep your paddings consistent. avoid being very specific. 
in this case -  even if you give `padding: 1.5rem`. it looks fine.

File: components/Auth/auth.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.auth-container{
+    margin: 4rem 1rem 2rem 1rem;
```
Comment (venkateshwarans) [inline components/Auth/auth.css]: try to keep the margins and padding consistent and avoid being very specific.

**Review by venkateshwarans — APPROVED** [review]
Just added a few suggestions. But overall everything looks great.

File: components/Auth/login.html (hunk 1 above)
Comment (VLeads) [reply components/Auth/login.html]: Changed names to lowercase.

File: components/Cart/cart.css (hunk 2 above)
Comment (VLeads) [reply components/Cart/cart.css]: Done. Thanks for the suggestion.

File: components/Auth/auth.css (hunk 3 above)
Comment (VLeads) [reply components/Auth/auth.css]: Done. 👍

File: components/Auth/signup.html [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+            </li>
+          </ul>
+        </div>
+      </div>
+      <p class="footer-rights">&copy; 2022 Rapid Store. All Rights Reserved</p>
+    </footer>
+
+    <script src="https://rapidui.netlify.app/js/main.js"></script>
+  </body>
+</html>
```
Comment (deekshithmd) [inline components/Auth/signup.html:156]: For me it is looking good to merge

Comment (deekshithmd) [inline components/Auth/signup.html:156]: For me it is looking good to merge

**Pull request merged**